roles/openshift_health_checker/openshift_checks/docker_storage_driver.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DockerStorageDriver(OpenShiftCheck):
+    """Check Docker storage driver compatibility.
+
+    This check ensures that Docker is using a supported storage driver,
+    and that Loopback is not being used (if using devicemapper).
```

Comment:
Written version of a mental note: this is another check (after memory and disk checks) that will annoy people creating toy clusters...